apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PartDEventTransformer.java

```diff
@@ -245,27 +246,43 @@ private static ExplanationOfBenefit transformClaim(PartDEvent claimGroup) {
             TransformerUtils.createExtensionQuantity(
                 CcwCodebookVariable.DAYS_SUPLY_NUM, claimGroup.getDaysSupply()));
 
-    // TODO CBBD-241 - This code was commented out because values other than
-    // "01"
-    // were coming thru
-    // such as "07". Need to discuss with Karl if this check needs to be
-    // here -
-
     /*
-     * if (claimGroup.serviceProviderIdQualiferCode == null ||
-     * !claimGroup.serviceProviderIdQualiferCode.equalsIgnoreCase("01"))
-     * throw new InvalidRifValueException(
-     * "Service Provider ID Qualifier Code is invalid: " +
-     * claimGroup.serviceProviderIdQualiferCode);
+     *   Code	    Code value
```

Comment:
This comment could use a header sentence introducing/explaining it, e.g. saying what CCW field the values you're listing are for.